website/docs/r/acm_certificate.html.markdown

```diff
@@ -143,7 +143,7 @@ The following arguments are supported:
 * Creating an Amazon issued certificate
     * `domain_name` - (Required) Domain name for which the certificate should be issued
     * `subject_alternative_names` - (Optional) Set of domains that should be SANs in the issued certificate. To remove all elements of a previously configured list, set this value equal to an empty list (`[]`) or use the [`terraform taint` command](https://www.terraform.io/docs/commands/taint.html) to trigger recreation.
-    * `validation_method` - (Required) Which method to use for validation. `DNS` or `EMAIL` are valid, `NONE` can be used for certificates that were imported into ACM and then into Terraform.
+    * `validation_method` - (Required for Amazon-issued certificate) Which method to use for validation. `DNS` or `EMAIL` are valid. This parameter must not be set for certificates that were imported into ACM and then into Terraform.
```

Comment:
I think this change would retain the necessary information while keeping with the standard format of just having `(Required)` or `(Optional)`.

If it feels more clear to mention the Amazon-issued bit, perhaps we could add it into the description itself. 

```suggestion
    * `validation_method` - (Optional) Which method to use for validation. `DNS` or `EMAIL` are valid. This parameter must not be set for certificates that were imported into ACM and then into Terraform.
```

Reply:
> I think this change would retain the necessary information while keeping with the standard format of just having `(Required)` or `(Optional)`.

I got the same feeling, I modified that, thanks 🙏